src/Pyz/Glue/GlueApplication/GlueApplicationDependencyProvider.php

```diff
@@ -699,6 +702,18 @@ protected function getApplicationPlugins(): array
         ];
     }
 
+    /**
+     * @deprecated Will be removed without replacement.
```

Comment:
Why is this marked as deprecated?

Reply:
@michbeck, it's old Glue infrastructure and is marked as deprecated in `suite-nonsplit`. However, it was discussed with @floriankraemer that since we still have it in `suite-nonsplit`, we should have it also in demo shops to have the same behaviour.